src/main/docs/index.md

```diff
@@ -50,6 +50,17 @@ Here is the simplest way to connect, query and disconnect
 {@link examples.Examples#gettingStarted()}
 ```
 
+You can also use environment variables to set default connection setting values, this is useful
+when you want to avoid hard-coding database connection information. 
+The following parameters are supported:
+
+* PGHOST
```

Comment:
use backticks for the values, i.e `PGHOST`

Reply:
Fixed in https://github.com/reactiverse/reactive-pg-client/pull/87/commits/bfbb7c37eadd99ebd74769a9121341b68ff36d32.